resources/puppetlabs/lein-ezbake/template/global/controller.sh.erb

```diff
@@ -131,7 +117,7 @@ params+=('--enterprise-build')
 <% end -%>
 
 # pull in rpm dependencies
-if [[ "$build_os" =~ el-?7 || "$build_os" =~ el-?6 || "$build_os" =~ sles || "$build_os" =~ fedora ]]; then
+if [[ "$os" = 'el' || "$os" = 'sles' || "$os" = 'fedora' ]]; then
```

Comment:
@demophoon does this work for you as simpler than hardcoding supported platform versions? This should only need updates if we add a new OS all together.